lint.go

```diff
@@ -1194,15 +1200,19 @@ func (f *file) lintErrorStrings() {
 		if s == "" {
 			return true
 		}
-		isCap, isPunct := lintCapAndPunct(s)
+		isCap, isPunct, isNewline := lintCapAndPunct(s)
 		var msg string
 		switch {
```

Comment:
This whole switch statement makes me sad...

Can we simply just say:
"error strings should not start capitalized or end with punctuation or a newline"
for all situations?